packages/http-client/src/http-client.spec.ts

```diff
@@ -94,6 +95,21 @@ describe('generic http client', () => {
     name: 'FORWARD-TOKEN-NOAUTH'
   };
 
+  function responseWithPublicKey(key: string = publicKey) {
```

Comment:
For reviewer, this was orginally imported from the verify test file, I just moved the function in here as I deleted the verify test file.